tests/fixtures.py

```diff
@@ -384,9 +385,28 @@ def federated_ursulas(ursula_federated_test_config):
 # Blockchain
 #
 
-@pytest.fixture(scope='session')
-def token_economics():
-    economics = StandardTokenEconomics()
+@pytest.fixture(scope='module')
+def token_economics(testerchain):
+
+    # Get current blocktime
+    blockchain = BlockchainInterfaceFactory.get_interface()
+    now = blockchain.w3.eth.getBlock(block_identifier='latest').timestamp
+
+    # Calculate instant start time
+    one_hour_in_seconds = (60 * 60)
+    start_date = now
+    bidding_start_date = start_date
+
+    # Ends in one hour
+    bidding_end_date = start_date + one_hour_in_seconds
+
+    economics = StandardTokenEconomics(
+        worklock_boosting_refund_rate=200,
+        worklock_commitment_duration=60*60,  # seconds
+        worklock_supply=NU.from_tokens(1_000_000),
```

Comment:
Can we have a constant/fixture for this figure? AFAIK, that's not the initial worklock supply, so it's a test figure.

Reply:
A fixture other than this one? OR so you just mean moving these test values to another module?